overlays/haskell-nix-extra/default.nix

```diff
@@ -40,4 +24,30 @@ in final: prev: with final; with lib; {
       chmod -R +w $out
       $STRIP $out/bin/*
     '' else p;
+
+  # Stamp one executable with version info.
```

Comment:
What does it mean to "stamp" an executable?

Reply:
providing the git rev to be printed with `--version`
(added some more doc)